src/clr-angular/utils/drag-and-drop/interfaces/draggable-event.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export interface ClrDraggableEvent {
```

Comment:
Is there a reason we couldn't just use the native `DragEvent`s?

Reply:
In our case, we are converting `MouseEvent` or `TouchEvent` into our custom drag event with more fields for custom ghost and its properties. So the API of our drag events will be almost completely different from the native DragEvent API.

Reply:
Still, Jeremy made me realize: Why isn't this called `ClrDragEvent`? It'd make more sense.

Reply:
Yes, that made me realize the same thing. I've updated that now.